src/microsoft/1.21/cbl-mariner2.0/Dockerfile

```diff
@@ -100,8 +100,9 @@ RUN set -eux; \
 		# To reduce image size, CBL-Mariner image doesn't include full ca-certificates: https://github.com/microsoft/CBL-Mariner/issues/3563
 		# Full certs are expected in a builder-style image (e.g. to fetch dependencies), so install them.
 		ca-certificates \
-		# Azure Pipelines expects "useradd", "groupadd", and "su".
+		# Azure Pipelines expects various tools that would be in a buildpack-deps based image.
```

Comment:
Same comment about linking as above

Reply:
For future reference: the `Dockerfile`s are generated, and the CI (`microsoft-go-images (Repeatability Test)`) checks that the Dockerfiles match what the templates evaluate to. So, it usually isn't necessary to check these files other than to make sure the template's logic did the right thing.

(If you noticed that the GitHub review UI hides these files by default, this is driven by https://github.com/microsoft/go-images/blob/microsoft/main/.gitattributes, which we inherited from upstream--along with the template system.)